bus-mapping/src/eth_types.rs

```diff
@@ -171,6 +171,57 @@ pub struct GethExecStep {
     pub storage: Storage,
 }
 
+impl From<GethExecStep> for GethExecStepInternal {
+    fn from(step: GethExecStep) -> Self {
+        GethExecStepInternal {
+            pc: step.pc,
+            op: step.op,
+            gas: step.gas,
+            gas_cost: step.gas_cost,
+            depth: step.depth,
+            stack: step
+                .stack
+                .0
+                .iter()
+                .map(|stack_elem| {
+                    DebugU256::from_big_endian(&stack_elem.to_be_bytes())
+                })
+                .collect(),
+            memory: step
+                .memory
+                .0
+                .iter()
+                .map(|mem_elem| {
+                    DebugU256::from_big_endian(&mem_elem.to_be_bytes())
+                })
+                .collect(),
+            storage: step
+                .storage
+                .0
+                .iter()
+                .map(|(k, v)| {
+                    (
+                        DebugU256::from_big_endian(&k.to_be_bytes()),
+                        DebugU256::from_big_endian(&v.to_be_bytes()),
+                    )
+                })
+                .collect(),
```

Comment:
```suggestion
            stack: step
                .stack
                .0
                .iter()
                .map(|stack_elem| DebugU256(stack_elem.0))
                .collect(),
            memory: step
                .memory
                .0
                .chunks(32)
                .map(|word| DebugU256::from_big_endian(word))
                .collect(),
            storage: step
                .storage
                .0
                .iter()
                .map(|(k, v)| (DebugU256(k.0), DebugU256(v.0)))
                .collect(),
```

A part from the simplification for stack and storage, notice the change in memory!  We need to go from Vec<u8> to Vec<DebugU256>.

Nevertheless this function is never called, but I still think it's good to fix it.

Reply:
Indeed! Thanks for the suggestion!